database/sql/src/main/scala/cromwell/database/slick/tables/MetadataEntryComponent.scala

```diff
@@ -62,6 +62,13 @@ trait MetadataEntryComponent {
     } yield metadataEntry).sortBy(_.metadataTimestamp)
   )
 
+  val metadataEntriesForWorkflowForArchiving = Compiled(
```

Comment:
[TOL] I mildly prefer functions to be named after what they do, not how they're used. Should make it easier to spot the difference between this and `metadataEntriesForWorkflowExecutionUuid`

```suggestion
  val metadataEntriesForWorkflowSortedById = Compiled(
```